app/util/GainsManip.js

```diff
@@ -0,0 +1,39 @@
+var LocalStorage = require('./LocalStorage');
+var fdialogs = require('node-webkit-fdialogs');
+var Logger = require('../util/Logger');
+var Commands = require('../models/Commands');
+var gains_config = require('../../config/gains-config');
+//Loads in configuration from config files into local storage if they dont exist. Subsequent
+var GainsManip = {
+    GainsExport: function () {
+        var keys=Object.keys(gains_config.default_settings);
+        var str;
+        var buf;
+        var exported_settings=gains_config.default_settings;
+        for (var i = 0; i < keys.length; i++)
+        {
+            exported_settings[keys[i]]=gains_config.get(keys[i]);
+        }
+        str=JSON.stringify(exported_settings,null,2);
+        buf=new Buffer(str);
+        fdialogs.saveFile(buf, function (err, path) {
+        });
+    },
+    GainsImport: function () {
+        var Dialog = new fdialogs.FDialog({
+            type: 'open',
+            accept: ['.txt'],
+            path: '~Documents'
+        });
+        fdialogs.readFile(function (err, content, path) {
+            var keys=Object.keys(gains_config.default_settings);
+            var object=JSON.parse(content,null,2);
+            for(var i=0;i<keys.length;i++)
+            {
+                gains_config.set(keys[i],object[keys[i]]);
```

Comment:
What happens if I try to import an old settings file and my default configuration has updated keys that dont exist within that file. Doing it this way will completely destroy (by setting it to undefined) the setting values that I originally had just because I imported an old settings file. You need to check first that object[keys[i]] exists with an if statement, before setting the gains_config. Or you could use the keys from the parsed file, with Object.keys(object) instead so theres no chance of you overwriting new settings.